docs/components/fasteners/nut.mdx

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
I didn't realize that we were adding all fasteners here. Does this duplicate any information that is already available somewhere else internally? Or is this intended to be the public facing version of your parts spreadsheet?

Reply:
This is a public facing version of our common parts. I think this information can help our customers quickly find replacement parts, or help with their integrations. We can also simplify our User Manuals by redirecting readers to the Components area of the website.

Should we keep the Components section of the website?

Reply:
I'm not opposed to sharing fasteners as long as it doesn't become a maintenance issue over time. If the list is fairly static, then it shouldn't be a problem.